docs/_docs/index.md

```diff
@@ -1,56 +1,47 @@
 ---
-title: Welcome
+title: 'Welcome'
 permalink: /docs/home/
 redirect_from: /docs/index.html
 ---
 
-This site aims to be a comprehensive guide to Jekyll. We’ll cover topics such
-as getting your site up and running, creating and managing your content,
-customizing the way your site works and looks, deploying to various
-environments, and give you some advice on participating in the future
-development of Jekyll itself.
+This site is a comprehensive guide to Jekyll. We’ll cover topics such as:
 
-## So what is Jekyll, exactly?
+  - Getting your site up and running
+  - Creating and managing your content
+  - Customizing how your site works and looks
+  - Deploying to various environments
```

Comment:
Used in this context, "various" is a '[weasel word][]'. Perhaps this line could be simply, "Deploying your site."

[weasel word]: http://matt.might.net/articles/shell-scripts-for-passive-voice-weasel-words-duplicates/